Service/OS/AndroidGCMNotification.php

```diff
@@ -57,14 +65,16 @@ class AndroidGCMNotification implements OSNotificationServiceInterface
     /**
      * Constructor
      *
-     * @param $apiKey
+     * @param bool         $dryRun
+     * @param string       $apiKey
      * @param bool         $useMultiCurl
      * @param int          $timeout
      * @param Logger       $logger
-     * @param AbstractCurl $client       (optional)
+     * @param AbstractCurl $client (optional)
      */
-    public function __construct($apiKey, $useMultiCurl, $timeout, $logger, AbstractCurl $client = null)
+    public function __construct($dryRun, $apiKey, $useMultiCurl, $timeout, $logger, AbstractCurl $client = null)
```

Comment:
ah - I see :) Yeah there is definitely some inconsistency in this bundle anyway, but am just aware of some people using the bundle outside of Symfony and instantiating the service itself, so keen to not break BC if possible that's all!